src/frontend/admin/src/components/templates/orders/view/OrderView.tsx

```diff
@@ -50,128 +65,148 @@ export function OrderView({ order }: Props) {
               />
-            </Alert>
-          </Grid>
-          <Grid xs={12} sm={6}>
-            <TextField
-              fullWidth={true}
-              disabled={true}
-              InputProps={{
-                endAdornment: getViewIcon(
-                  PATH_ADMIN.organizations.edit(order.organization.id),
-                ),
-              }}
-              label={intl.formatMessage(orderViewMessages.organization)}
-              value={order.organization.title}
-            />
-          </Grid>
-          <Grid xs={12} sm={6}>
-            <TextField
-              fullWidth={true}
-              disabled={true}
-              InputProps={{
-                endAdornment: getViewIcon(
-                  PATH_ADMIN.products.edit(order.product.id),
-                ),
-              }}
-              label={intl.formatMessage(orderViewMessages.product)}
-              value={order.product.title}
-            />
-          </Grid>
-          <Grid xs={12}>
-            {order.course && !order.enrollment && (
+            </Grid>
+            <Grid xs={12} sm={6}>
               <TextField
                 fullWidth={true}
                 disabled={true}
                 InputProps={{
                   endAdornment: getViewIcon(
-                    PATH_ADMIN.courses.edit(order.course.id),
+                    PATH_ADMIN.products.edit(order.product.id),
                   ),
                 }}
-                label={intl.formatMessage(orderViewMessages.course)}
-                value={order.course.title}
+                label={intl.formatMessage(orderViewMessages.product)}
+                value={order.product.title}
               />
-            )}
-          </Grid>
-          <Grid xs={12}>
-            {order.enrollment && (
+            </Grid>
+            <Grid xs={12}>
+              {order.course && !order.enrollment && (
+                <TextField
+                  fullWidth={true}
+                  disabled={true}
+                  InputProps={{
+                    endAdornment: getViewIcon(
+                      PATH_ADMIN.courses.edit(order.course.id),
+                    ),
+                  }}
+                  label={intl.formatMessage(orderViewMessages.course)}
+                  value={order.course.title}
+                />
+              )}
+            </Grid>
+            <Grid xs={12}>
+              {order.enrollment && (
+                <TextField
+                  fullWidth={true}
+                  disabled={true}
+                  multiline={true}
+                  label={intl.formatMessage(orderViewMessages.enrollment)}
+                  value={`${order.enrollment.course_run.title} (${
+                    order.enrollment.course_run.state?.text
+                  })\n${intl.formatMessage(
+                    orderViewMessages.enrollmentAt,
+                  )}: ${new Date(
+                    order.enrollment.created_on,
+                  ).toLocaleDateString()}`}
```

Comment:
IMO, you should create a message for this one. It will clearer and according to the language, maybe the date should be placed anywhere else.